frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts

```diff
@@ -0,0 +1,31 @@
+// import {ImageStyle} from "~/components/entry/admin/images/meta/lists/image-style";
```

Comment:
on the file itself - there is no `imagelabel` entry, should be in `admin/images/meta/`